Changelog.md

```diff
@@ -15,6 +15,7 @@
 - [#609]: Added `Writer::write_serializable` to provide the capability to serialize
   arbitrary types using serde when using the lower-level `Writer` API.
 - [#615]: Added ability to set entity resolver when deserialize using borrowing reader.
+- [#620]: Added ability to enforce the expansion of empty elements.
```

Comment:
I prefer to use link to the issue when PR fixes/implements an issue

Reply:
Fixed in 5250849—did you see my other comment?